src/leiningen/i18n/utils.clj

```diff
@@ -0,0 +1,11 @@
+(ns leiningen.i18n.utils
+  "Plugin for i18n tasks. Start by using i18n init"
+  (:require [clojure.string :as cstr]))
+
+(defn replace-first-in-multiline
+  "Replace first instance of regex-match in the passed in string with the
+   replacement. Uses RegExp's m flag (?m) so that ^/$ matches start/end of line."
+  [s match-string replacement]
```

Comment:
The docstring and the arguments should be updated to reflect that the function accepts a regex rather than a string. It should probably also mention that the regex need only match the beginning of the line (although I'd prefer that this function not add the `.*` before the `$` so the behavior is more apparent at the call site and it's possible for the caller to use a stricter match if they prefer).